### DISCUSSION TITLE
https://www.hammerdb.com/document.html is down

### DISCUSSION THREAD

**Discussion by SQLAmanda** [TPC-Council/HammerDB#221] [Q&A] [answered]
502 Bad Gateway
![MicrosoftTeams-image (8)](https://user-images.githubusercontent.com/56175477/112852505-9fb3d880-9071-11eb-9d9d-584166b71341.png)

**Comment by sm-shaw** [comment]
Yes this is a known issue with the web service provider - They are working on it, status can be checked here https://help.fasthosts.co.uk/app/systemstatus

**Comment by sm-shaw** [comment, marked as answer]
This is now resolved. For future reference the link https://help.fasthosts.co.uk/app/systemstatus can be used to check the website status for known issues.
